packages/components/src/mobile/notification-sheet/index.native.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * External dependencies
+ */
+import { Platform, View, Text } from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { BottomSheet, Icon } from '@wordpress/components';
+import { withPreferredColorScheme } from '@wordpress/compose';
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import styles from './style.scss';
+
+function NotificationSheet( { title, getStylesFromColorScheme, isVisible, onClose, type = 'singular', withMessage = true, withQuotes = true } ) {
+	const infoTextStyle = getStylesFromColorScheme( styles.infoText, styles.infoTextDark );
+	const infoTitleStyle = getStylesFromColorScheme( styles.infoTitle, styles.infoTitleDark );
+	const infoDescriptionStyle = getStylesFromColorScheme( styles.infoDescription, styles.infoDescriptionDark );
+	const infoSheetIconStyle = getStylesFromColorScheme( styles.infoSheetIcon, styles.infoSheetIconDark );
+
+	// translators: %s: Name of the block
+	const titleFormatSingular = Platform.OS === 'android' ? __( '%s isn\'t yet supported on WordPress for Android' ) :
+		__( '%s isn\'t yet supported on WordPress for iOS' );
+
+	const titleFormatPlural = Platform.OS === 'android' ? __( '%s aren\'t yet supported on WordPress for Android' ) :
+		__( '%s aren\'t yet supported on WordPress for iOS' );
+
+	const titleFormat = type === 'plural' ? titleFormatPlural : titleFormatSingular;
```

Comment:
Plurals and localization is trickier than this, and currently not well supported at least on iOS.

Not only there can be [multiple plural forms](https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_and_Plurals), but whether the title is plural or not will depend on the language and shouldn't be passed as a prop. "Color Settings" is plural in English, but it might not be in a different translation.

I'd stick to the singular form and maybe try to have the copy be more agnostic of plurals: `WordPress for XXX doesn't support %s yet`

Reply:
Do we want to have the same message for missing feature/functionality/component? I'm totally fine with your idea of a generic message, @iamthomasbishop what's your opinion on that? 

However, let's wait with refactor that since there is an ongoing [discussion](https://github.com/wordpress-mobile/gutenberg-mobile/issues/1513#issuecomment-571123862) 
how do we want to present that message. Maybe I will have to revert these changes once we decide to discard the sheet concept.

Reply:
A generic message is fine. My only request is if we change it let's make sure it gets a thorough review.

Reply:
Outdated - `NotifcationSheet` is removed since we are not using bottom sheet to inform user about missing theme colors.